release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2919,6 +2919,12 @@ It is anticipated that an upcoming z-stream release will include a fix for this
 +
 To work around this issue: If your control plane nodes are schedulable, and the pods can run on worker nodes, use `nodeSelector` or Affinity to schedule the pod in worker nodes.(link:https://issues.redhat.com/browse/OCPBUGS-18581[*OCPBUGS-18581*])
 
+* Currently, the update to HAProxy version 2.6.13 in {product-title} {product-version} causes an increase in P99 latency for re-encrypt traffic. This is observed when the volume of ingress traffic puts the HAProxy component of the `IngressController` custom resource (CR) under a considerable load. The latency increase does not affect overall throughput, which remains consistent.
++
+The default `IngressController` CR is configured with 4 HAProxy threads. If you experience elevated P99 latencies during high ingress traffic conditions, specifically with re-encrypt traffic, it's recommended to increase the number of HAProxy threads to reduce latency. (link:https://issues.redhat.com/browse/OCPBUGS-18936[*OCPBUGS-18936*])
+
+* For {sno-caps} on {product-version} and Google Cloud Platform (GCP), there is a known issue with the Cloud Network Config Controller (CNCC) entering a `CrashLoopBackOff` state. This occurs at initialization time when the CNCC tries to reach the Azure internal load balancer address and the resulting hairpin traffic is not correctly prevented in OVN-Kubernetes shared gateway mode on GCP causing it to get dropped. Cluster Network Operator will show a `Progressing=true` status in such case. Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-20554[*OCPBUGS-20554*])
```

Comment:
Just want to verify that we should be mentioning Azure here. Only asking because we introduce the issue as single-node openshift on GCP, but then reference an Azure internal load balancer. Could be my complete lack of knowledge about this part of the product, but wanted to raise it just in case.

Reply:
Ops. There's no azure here, my bad!
I had this other bug in mind (https://issues.redhat.com/browse/OCPBUGS-9972), which is the same problem we are describing here for GCP, but on Azure. Sorry :(

Reply:
So it'll be `GCP internal load balancer`